oap-server/server-bootstrap/src/main/resources/component-libraries.yml

```diff
@@ -421,3 +441,7 @@ Component-Server-Mappings:
   pulsar-consumer: Pulsar
   rest-high-level-client: Elasticsearch
   mariadb-jdbc: Mariadb
+  cURL: HttpClient
```

Comment:
This mapping should be targeting server, `HttpClient` isn't the case, I think. Why do you need this?

Reply:
Because this is an `HttpClient` toolkit

Reply:
It doesn't represent the client type, it is for the server side. Such as, once you have a client exit span, which type of the server node should be shown in the topology page.